### PR TITLE
fix: ensure nfpm can run on windows and still build correct rpm, deb, and apk packages

### PR DESCRIPTION
**Pull request by djgilcrease**
- [x] Add testing on windows to the actions
- [x] Figure out why the rpm tests cannot remove the tmp rpm files created

### CODE REVIEW

**Comment by caarlos0** [comment]
LGTM so far, once this is fixed/released, will do the same on goreleaser.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -9,6 +9,27 @@ on:
...
+    runs-on: windows-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.14
```
Comment (caarlos0) [inline .github/workflows/build.yml:22]: should be 1.15?

```suggestion
          go-version: 1.15
```

File: go.mod [hunk 2]
```diff
@@ -3,22 +3,44 @@ module github.com/goreleaser/nfpm
 go 1.14
 
 require (
+	github.com/Djarvur/go-err113 v0.1.0 // indirect
```
Comment (caarlos0) [inline go.mod:6]: missing a `go mod tidy`?

File: internal/files/files.go [hunk 3]
```diff
@@ -43,3 +44,9 @@ func Expand(filesSrcDstMap map[string]string, disableGlobbing bool) ([]FileToCop
 	})
 	return files, nil
 }
+
+// ToNixPath converts the path to the nix style path
+// Windows style path separators are escape characters so cause issues.
```
Comment (caarlos0) [inline internal/files/files.go:49]:
```suggestion
// ToNixPath converts the given path to a nix-style path.
//
// Windows-style path separators are considered escape 
// characters by some libraries, which can cause issues.
```

**Review by caarlos0 — COMMENTED** [review]
the changes in the www/docs folder seem unrelated 🤔

File: go.mod (hunk 2 above)
Comment (djgilcrease) [reply go.mod:6]: even after a `make deps` and an explicit `go mod tidy` this still shows up

**Review by caarlos0 — APPROVED** [review]

**Pull request merged**